### PR TITLE
Python strings .rstrip(), Closes #3363

### LINKED ISSUE #3363: [Term Entry] Python Strings .rstrip()
### Reason/inspiration (optional)

We would like a new term entry on the `.rstrip()` method for Python: [Strings](https://www.codecademy.com/resources/docs/python/strings). The entry should go in a new file under `docs/content/python/concepts/strings/terms/rstrip/rstrip.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```py `code snippet.
- A “Codebyte Example” section with a ` ```codebyte/python ` snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by reganmarie**
### Description

- Closes issue #3363 
- Created rstrip.md on filepath docs/content/python/concepts/strings/terms/rstrip/rstrip.md
- Added Title, Description, Subjects, Tags, and Catalog Content
Added description, syntax, example, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines
-Checked for spelling and grammar errors
- Read for clarity and readability

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/rstrip/rstrip.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- `string` (required): The string to remove trailing characters from.
+- `characters` (optional): The characters to remove from the end of the string. All combinations of these characters will be removed from the end of the string until there is a mismatch. If characters are not provided, trailing whitespace will be removed from the string by default.
+
+## Example
+
+In the following example, the `rstrip()` method is called twice.
+
+The first call does not specify characters to remove, so whitespace is removed.
+
+The second call specifies the characters to remove from the end of the string.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rstrip/rstrip.md:32]:
```suggestion
In the following example, the `.rstrip()` method is called twice. The first call does not specify characters to remove, so whitespace is removed. And the second call specifies the characters `"se"` to be removed from the end of the string.
```

File: content/python/concepts/strings/terms/rstrip/rstrip.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+This will result in the following output:
+
+```shell
+Example A: happiness
+Example B: Tenn
+```
+
+## Codebyte Example
+
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rstrip/rstrip.md:53]: there should be an introductory sentence here

File: content/python/concepts/strings/terms/rstrip/rstrip.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+```codebyte/python
+first_string = "  unreal   unearth   "
+first_result = first_string.rstrip()
+print(first_result)
+
+second_string = "Francesca"
+second_result = second_string.rstrip("sca")
+print(second_result)
+``
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rstrip/rstrip.md:62]:
```suggestion
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@reganmarie I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by reganmarie** [comment]
@caupolicandiaz Updated with the suggested changes! 
- Condensed the example introduction in line 28 to the suggested change.
- Added an introductory sentence to the Codebyte example:
In the following example, the `.rstrip()` method is called twice. The first call does not specify characters to remove, so whitespace is removed. The second call specifies the characters `"sca"` to be removed from the end of the string.

**Review by caupolicandiaz — APPROVED** [review]
@reganmarie This is ready for a second review.

**Comment by reganmarie** [comment]
Thank you @caupolicandiaz! 😊

**Review by yangc95 — APPROVED** [review]

**Pull request merged**